examples/resources/alkira_connector_azure_er/resource.tf

```diff
@@ -0,0 +1,25 @@
+resource "alkira_connector_azure_er" "test_basic" {
+    name =              "AzureErName"
+    size =              "LARGE"
+    enabled =           true
+    vhub_prefix =       "10.129.0.0/23"
+    cxp =               "USWEST-AZURE-1"
+    tunnel_protocol =   "VXLAN_GPE"
+    group =             alkira_group.tftest.name
+
+    # You can add more instances blocks.
+    instances {
+        name =                      "instance13"
```

Comment:
you probably want to run `terraform format` on this example to make it aligned properly.